lib/consumerGroup.js

```diff
@@ -53,6 +53,15 @@ function ConsumerGroup (memberOptions, topics) {
   const self = this;
   this.options = _.defaults((memberOptions || {}), DEFAULTS);
 
+  if (!this.options.heartbeatInterval) {
+    this.options.heartbeatInterval = Math.floor(this.options.sessionTimeout / 3);
```

Comment:
According to offical kafka doc for `heartbeat.interval.ms`

> The expected time between heartbeats to the group coordinator when using Kafka's group management facilities. Heartbeats are used to ensure that the worker's session stays active and to facilitate rebalancing when new members join or leave the group. The value must be set lower than session.timeout.ms, but typically should be set no higher than 1/3 of that value. It can be adjusted even lower to control the expected time for normal rebalances.